README.md

```diff
@@ -219,7 +220,8 @@ When the are significant changes to content structure or styling it may be neces
 [production]: https://eyfs-covid-recovery.london.cloudapps.digital
 [staging]: https://ey-recovery-staging.london.cloudapps.digital
 [prototype-repo]: https://github.com/DFE-Digital/ey-recovery-prototype
-[prototype-app]: https://eye-recovery.london.cloudapps.digital
+[prototype-app]: https://eye-recovery.herokuapp.com
```

Comment:
is the heroku app back?

Reply:
@ebrett yes, only the interim version is on gpaas